### PR TITLE
Add container cluster classes

### PR DESCRIPTION
**Pull request by ehankland**

### CODE REVIEW

File: perfkitbenchmarker/container_service.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+FLAGS = flags.FLAGS
+_CLUSTER_REGISTRY = {}
+
+flags.DEFINE_string('kubeconfig', None,
+                    'Path to kubeconfig to be used by kubectl')
+
+flags.DEFINE_string('kubectl', 'kubectl',
+                    'Path to kubectl tool')
+
```
Comment (yuyantingzero) [inline perfkitbenchmarker/container_service.py]: too many empty lines

File: perfkitbenchmarker/providers/gcp/google_container_engine.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    super(GkeCluster, self).__init__(spec)
+    self.name = 'pkb-%s' % FLAGS.run_uri
+    self.project = spec.vm_spec.project
+
+  def _Create(self):
+    """Creates the cluster."""
+    cmd = util.GcloudCommand(self, 'container', 'clusters', 'create', self.name)
+    cmd.flags['num-nodes'] = self.num_nodes
+    cmd.flags['machine-type'] = self.machine_type
+    cmd.flags['cluster-version'] = '1.6.2'
```
Comment (yuyantingzero) [inline perfkitbenchmarker/providers/gcp/google_container_engine.py]: should cluster-version be a constant or maybe a flag?

File: perfkitbenchmarker/container_service.py (hunk 1 above)
Comment (ehankland) [inline perfkitbenchmarker/container_service.py]: oops

File: perfkitbenchmarker/providers/gcp/google_container_engine.py (hunk 2 above)
Comment (ehankland) [inline perfkitbenchmarker/providers/gcp/google_container_engine.py]: made it a constant

**Pull request merged**